src/main/proto/halo_cmm/uk/pilot/common_template.proto

```diff
@@ -27,4 +27,27 @@ message Common {
     display_name: "Common Template"
     description: "A common base template for Origin Event filtering."
   };
+
+  enum Sex {
```

Comment:
this was requested by ISBA/Origin's I&D team based on the policies as what we are referring to is the viewer's sex rather than gender (male, female, transgender, gender neutral, non-binary, agender, pangender, genderqueer etc...)  - I guess the mapping to internal systems shouldn't cause much problem ?

Reply:
I see, it's ok.